### ISSUE TITLE
Read configurations from different files

### ISSUE THREAD

**Issue by VincenzoFerme** [benchflow/data-analyses-scheduler#21]
We are moving some configuration variables to be handled by Consul, and serialised to file by envconsul. This means we need to read also another `*.yml` file (for now at boot time of the application, since we do not expect it to be updated), and store the information in a struct.

Location of the file in the container:
- /app/config.yml

A reference to start with for reading information also from this file:
- https://github.com/go-yaml/yaml

A reference for future work, where we should monitor the changes to the file and update the configuration stored in memory:
- https://github.com/go-fsnotify/fsnotify
- https://godoc.org/golang.org/x/exp/inotify

Once it is done, remove the MINIO variables also from the deployment descriptor of the service.
